src/hyperfine/export/mod.rs

```diff
@@ -15,10 +15,10 @@ use hyperfine::internal::Second;
 #[derive(Debug, Default, Clone, Serialize)]
 pub struct ExportEntry {
```

Comment:
If this is now used in other modules, maybe we should move it to `internal.rs` or a new `types.rs` module. I think it would also make sense to rename it to `BenchmarkResult` or something similar.

Reply:
Makes sense. If making a dedicated ```types``` module, we can move the other structs into it also and just leave internal functionality in ```internal.rs```